docs/app/Examples/elements/Segment/Groups/SegmentHorizontalSegmentsExample.js

```diff
@@ -4,11 +4,11 @@ import { Segment } from 'stardust'
 export default class SegmentHorizontalSegmentsExample extends Component {
   render() {
     return (
-      <Segment.Segments className='horizontal'>
+      <Segment.Group aligned='horizontal'>
```

Comment:
Amended to horizontal:bool, disregard comment asking whether `vertical` would be supported.